dev/content/Forms.vue

```diff
@@ -204,20 +207,31 @@ const toggleProps = [{ name: "modelValue", required: true, type: "string" }]
           </label>
           <div class="mt-1">
             <BaseInput
+              class="xy-input-error"
```

Comment:
I notice the addition here and deletion in `index.css` Is the idea to use the `error` prop instead? Asking more from a usage standpoint and groking the changes rather than a nitpick.

Reply:
Right, the intention is to set the error prop instead of using the class.  That was just left behind in the docs, but is cleaned up now.  The class based styling didn't correctly apply the styles it defined to begin with and had limited utility in general as it was really only suitable for text like inputs.